src/Glpi/Form/Destination/CommonITILField/ValidationField.php

```diff
@@ -218,7 +220,17 @@ public function prepareInput(array $input): array
             ];
             $actors = [];
 
-            foreach ($input[$this->getKey()][ValidationFieldConfig::SPECIFIC_ACTORS] as $actor) {
+            foreach ($input[$this->getKey()][ValidationFieldConfig::SPECIFIC_ACTORS] as $key => $actor) {
+                if (
+                    in_array($key, array_keys($available_actor_types))
+                    && is_array($actor)
+                    && ctype_digit(implode($actor))
+                ) {
+                    $itemtype = $key;
+                    $actors[$itemtype] = $actor;
+                    continue;
+                }
```

Comment:
Is this necessary to introduce another input format just for this migration? It will have to be maintained and, as it is not even documented that is has been made for this purpose, it will be hard to now, in a few month/years, if it still used or not.